tools/mmp/driver.cs

```diff
@@ -1080,6 +1080,11 @@ static string GenerateMain ()
 				if (App.EnableDebug)
 					sw.WriteLine ("\txamarin_debug_mode = TRUE;");
 
+				if (App.EnableSGenConc)
```

Comment:
Does desktop AOT-compiled code have to be compiled with the same MONO_GC_PARAMS as is used at runtime?